joomla/joomla-cms
# [4.1] Add server timing response header in debug mode

### Summary of Changes
Browsers do support a [server timing](https://www.w3.org/TR/server-timing/#the-server-timing-header-field.) header field which adds information what happens on the server side. This basically gives the developer insight what happens in the "Time to First Byte".
This pr adds the missing header in the debug plugin, as this header should not be added when in production mode because of performance reasons.

Would be nice to get some feedback about spicing up the values of the header as currently I'm just logging them as they are.

The view looks like in the following picture:
![image](https://user-images.githubusercontent.com/251072/144827604-83277df5-668a-4a38-acce-ad22e0bdc0ca.png)

### Testing Instructions
- When testing in Firefox, make sure you open the page with HTTPS
- Put joomla in debug mode
- In the debug plugin with make sure that the Profile collector is enabled (is by default)
- Open any page
- Open the dev tools and check the Network tab
- Select the first entry and then click on the timings tab of the specific request

### Actual result BEFORE applying this Pull Request
Only the request timing information is shown.

### Expected result AFTER applying this Pull Request
Beside the request timing is the server timing information shown.
